pruner/run.sh

```
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+set -e
+
+SCRIPT_DIR=$(cd $(dirname $0) ; pwd -P)
+
+TASK=$1
+ARGS=${@:2}
+
+
+help__run="run <..args> : run run"
```

Shellcheck (0.11.0)

[warning] help__run appears unused. Verify use (or export if used externally).

(SC2034)